pkg/standalone/standalone.go

```diff
@@ -1013,3 +1049,37 @@ func parseVersionFile(fromDir string) (string, string) {
 
 	return versions[daprRuntimeFilePrefix], versions[dashboardFilePrefix]
 }
+
+func getDefaultRegistry() string {
+	imageRegistryName := githubContainerRegistryName
+	if val := os.Getenv("DEFAULT_IMAGE_REGISTRY_MODE"); val != "" {
```

Comment:
os.LookupEnv() returns val and bool for if exists ...
should this be `DAPR_DEFAULT_IMAGE_REGISTRY` ? since all other env vars in CLI generally have a `DAPR` prefix ...

Reply:
https://pkg.go.dev/os#LookupEnv